### PR TITLE
[ISV-993][ISV-1144] Publish bundle to index using IIB

### PR DESCRIPTION
**Pull request by Allda**
Use "index" CLI to publish operator bundle to operator index.

JIRA: ISV-993

### CODE REVIEW

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ spec:
       description: Repository name assigned to certification project from Red Hat Connect
     - name: dist_method
```
Comment (MarcinGinszt) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml:22]: This is no longer used and can be removed

File: README.md [hunk 2]
```diff
@@ -145,5 +145,7 @@ tkn pipeline start operator-release-pipeline \
   --workspace name=image-data,volumeClaimTemplateFile=templates/workspace-template-small.yml \
   --workspace name=pyxis-ssl-credentials,secret=operator-pipeline-api-certs \
   --workspace name=github-bot-token,secret=github-bot-token \
+  --workspace name=kerberos-keytab,secret=kerberos-keytab \
+  --workspace name=iib-quay-credentials,secret=iib-quay-credentials \
```
Comment (MarcinGinszt) [inline README.md:149]: I think this workspace is not necessary

**Review by MarcinGinszt — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (kosciCZ) [reply README.md]: Good catch

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml (hunk 1 above)
Comment (kosciCZ) [reply ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml]: Removed

**Review by amisstea — APPROVED** [review]
LGTM :+1:

**Comment by kosciCZ** [comment]
To summarize changes that go above the original AC:
- uses IIB batch api
- uses own Quay credentials for prod and stage
- fixes ISV-1144

**Pull request merged**